addOns/help/src/main/javahelp/contents/releases/2_8_0.html

```diff
@@ -9,6 +9,50 @@
 <BODY BGCOLOR="#ffffff">
 <H1>Release 2.8.0</H1>
 
+This is a bug fix and enhancement release, which requires a minimum of Java 8. 
+Note that a minimum of Java 11 is recommended, especially for high DPI displays.
```

Comment:
Did we change something related to hDPI support?

Are we concerned about this recent OpenJDK 12 report? https://github.com/zaproxy/zaproxy/issues/5368

Reply:
We should not need to change anything (I think), that's provided by the JRE: http://openjdk.java.net/jeps/263

This issue seems related: https://sourceforge.net/p/launch4j/bugs/197 we might need to tweak the launch4j configs.

Reply:
Ok, thanks!